row_store.go

```diff
@@ -23,21 +23,24 @@ import (
 	"github.com/getlantern/goexpr"
 	"github.com/getlantern/wal"
 	"github.com/getlantern/zenodb/bytetree"
+	"github.com/getlantern/zenodb/common"
 	"github.com/getlantern/zenodb/core"
 	"github.com/getlantern/zenodb/encoding"
 )
 
 const (
 	// File format versions
 	FileVersion_4      = 4
-	CurrentFileVersion = FileVersion_4
+	FileVersion_5      = 5
```

Comment:
We introduce a new file version because the header now includes not one offset, but multiple offsets per source (leader).